web/assets/handle-create-document-button.js

```diff
@@ -0,0 +1,33 @@
+const handleCreateDocumentButton = () => {
+    /*  After creating a non case contact recipient, if a user clicks the create document button without selecting a
+        recipient, the recipient they just created will be lost (as non case contacts are not retrieved as part of the
+        call to get recipients). So JS is required to disable the button and enable it if at least one recipient has
+        been selected */
+
+  /** @type HTMLButtonElement|null createDocumentButton */
+  const createDocumentButton = document.querySelector(
+    '[data-module="create-document-button"]'
+  );
```

Comment:
This indenting it inconsistent with other lines, which suggests your don't have Prettier working? Worth sorting out to avoid future conflicts.

Reply:
Sorted, and i've fixed the formatting of some other files I worked in/created too